k8s/etcd/README.md

```diff
@@ -18,18 +17,17 @@ Popular open stacks on Kubernetes packaged by Google.
 
 ![Architecture diagram](resources/etcd-k8s-app-architecture.png)
 
-A Kubernetes StatefulSet manages all etcd Pods in this application. Each
-Pod runs a single instance of etcd. Each instance of etcd communicates each other with peer port 2380. Connection between cluster members is automatically encrypted with TLS certificates.
-TLS certificates are created and used by each instance of etcd when starting cluster service.
+A Kubernetes StatefulSet manages all of the etcd Pods in this application. Each Pod runs a single instance of etcd, and each instance of etcd communicates with the others by using peer port 2380. Connections between cluster members are automatically encrypted with TLS certificates.
+
+Each instance of etcd generates its own TLS certificate at the time that cluster service is started.
```

Comment:
@farajn9, @khajduczenia 

Do we generate the certificate (I cannot see it into the `schema.yaml`? And it's just one certificate or a few ones?